hotrod/src/main/java/io/hyperfoil/hotrod/steps/HotRodRequestBuilder.java

```diff
@@ -0,0 +1,59 @@
+package io.hyperfoil.hotrod.steps;
+
+import java.util.Arrays;
+import java.util.List;
+
+import org.kohsuke.MetaInfServices;
+
+import io.hyperfoil.api.config.Name;
+import io.hyperfoil.api.config.Step;
+import io.hyperfoil.api.config.StepBuilder;
+import io.hyperfoil.core.builders.BaseStepBuilder;
+import io.hyperfoil.core.session.SessionFactory;
+import io.hyperfoil.hotrod.api.HotRodOperation;
+import io.hyperfoil.hotrod.resource.HotRodResourceKey;
+
+/**
+ * Issues a HotRod request and registers handlers for the response.
+ */
+@MetaInfServices(StepBuilder.class)
+@Name("hotrodRequest")
+public class HotRodRequestBuilder extends BaseStepBuilder<HotRodRequestBuilder> {
+
+   private HotRodOperationBuilder operation;
+   private HotRodCacheNameBuilder cacheName;
+   private int stepId = 1;
+
+   @Override
+   public List<Step> build() {
+
+      HotRodResourceKey key = new HotRodResourceKey();
+      HotRodRequestStep step = new HotRodRequestStep(stepId, key, operation.build(), cacheName.build(),
+            SessionFactory.access("cacheKey"), SessionFactory.access("cacheValue"));
+      HotRodResponseStep secondHotRodStep = new HotRodResponseStep(stepId, key);
+      return Arrays.asList(step, secondHotRodStep);
+   }
+
+   public HotRodRequestBuilder cacheName(String cacheName) {
```

Comment:
What can I do in order to see the documentation locally?

Reply:
When you run `mvn install` it will create the markdown-docs in `distribution/target/steps/`. You can clone `https://github.com/Hyperfoil/hyperfoil.github.io` and copy contents of the generated folder into `docs/steps/` there and run `bundle exec jekyll serve` to build the web locally.

Reply:
Under `distribution/target/steps/`, there is nothing related to `HotRod` even when I did
```
   /**
    * A named cache from the remote server if the cache has been defined, otherwise if the cache name is undefined,
    * it will return null.
    *
    * @param cacheName name of cache to retrieve
    * @return
    */
   public HotRodRequestBuilder cacheName(String cacheName) {
      return cacheName(() -> new HotRodCacheNameBuilder.Provided(cacheName));
   }
```

Reply:
Sorry, you probably need to add hotrod here: https://github.com/Hyperfoil/Hyperfoil/blob/62329a6288436cde1d5e31181925801c7ba67008/distribution/pom.xml#L193

Reply:
Done

Reply:
The `put()` and `operation()` should get javadoc as well, especially that is what you expect your users to use.